pkgs/by-name/so/softether/package.nix

```diff
@@ -1,22 +1,23 @@
 {
   lib,
   stdenv,
-  fetchurl,
+  fetchFromGitHub,
   openssl,
   readline,
   ncurses,
   zlib,
   dataDir ? "/var/lib/softether",
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "softether";
-  version = "4.38";
-  build = "9760";
+  version = "v4.41-9782-beta";
```

Comment:
You can remove the v and put it into the rev for the fetch.

Reply:
ack